tools/generate_tables_sql_and_testNames.php

```diff
@@ -1,118 +1,234 @@
 #!/usr/bin/php
 <?php
-
 /**
- * generate_tables_sql.php takes the ip_output.txt file generated from
- * quickform_parser.php and outputs an sql build file for the table of each
- * instrument it finds in the ip_output.txt file.  These sql files are output
- * to the tables_sql/ subdirectory.
+ * Behavioural
+ *
+ * PHP Version 7
  *
- * ex cmd:  php generate_tables_sql.php
+ * Parses a LINST file and creates a corresponding SQL script. Optionally 
+ * applies this new script automatically.
  *
- * @package behavioural
+ * @category Tools
+ * @package  Behavioural
+ * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
  */
-
-//Ensure php version compatability
-//taken from php.net notes
-
-// require all relevant OO class libraries
-require_once __DIR__ . "/../vendor/autoload.php";
-require_once "../php/libraries/Database.class.inc";
-require_once "../php/libraries/NDB_Config.class.inc";
-require_once "../php/libraries/NDB_BVL_Instrument.class.inc";
-
-$data = stream_get_contents(STDIN);
-
-$instruments=explode("{-@-}",trim($data));
-
-$tblCount=0;
-$parameterCount=0;
-$pages = array();
-foreach($instruments AS $instrument){
-    $catId="";
-    $items=explode("\n",trim($instrument));
-    foreach($items AS $item){
-        $paramId="";
-        $bits=explode("{@}",trim($item));
-        if(preg_match("/Examiner[0-9]*/" , $bits[1])){
-            continue;
-        }
-        switch($bits[0]){
-            //generate the CREATE TABLE syntax
+require_once 'generic_includes.php';
+// Display usage message if flag parameter is absent or invalid.
+if (!isset($argv[2]) 
```

Comment:
this is not consistent in any way, but I've been using `tosql` and `confirm` instead of `--to-sql` and `--apply` we might want to consider documenting a standard at some point and using it in all the tools

Reply:
I can change these to `tosql` and `confirm`. No need for this to be different.